modules/cmpapi/.mocharc.json

```diff
@@ -32,9 +32,8 @@
     "./test/encoder/section/UsCtV1.test.ts",
     "./test/encoder/section/UsNatV1.test.ts",
     "./test/encoder/section/UsUtV1.test.ts",
-    "./test/encoder/section/UspV1.test.ts",
-    "./test/encoder/section/UspV1.test.ts",
     "./test/encoder/section/UsVaV1.test.ts",
+    "./test/encoder/section/UspV1.test.ts",
```

Comment:
will this create a merge conflict with #54?

Reply:
Resolved